### PR TITLE
Modify vim cheatsheet

### PR DESCRIPTION
**Pull request by NaveenKarippai**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [ ] New Instant Answer
    - [ ] Cheat Sheets: **`New {Cheat Sheet Name} Cheat Sheet`**
    - [ ] Other: **`New {IA Name} Instant Answer`**
- [x] Improvement
    - [ ] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
    - [x] Enhancement: **`vim_cheat_sheet: added new section, modified commands`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**

###### Description of changes

Some day-to-day commands in vim like screen movement section and
window-related subcommands were not added in the cheatsheet.

Provide an overview of the changes this pull request introduces.

* description of cheatsheet modified

* screen movement section added

* window-related and other subcommands added/modified

###### Which issues (if any) does this fix?

Fixes #NNNN - how specifically does it fix the issue?

###### People to notify (@GuiltyDolphin)

---

Instant Answer Page: https://duck.co/ia/view/vim_cheat_sheet

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @kablamo

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by jophab** [comment]
Can we have section names as follows? 
  
         "Exiting",
          "Cursor Movement",
         "Screen Movement",
          "Insert Mode - Inserting/Appending Text",
          "Editing",
          "Marking Text (Visual Mode)",
          "Search and Replace",
          "Working with Multiple Files",
          "Tabs",
      "Miscellaneous",
      "Basic Configuration"

**Comment by NaveenKarippai** [comment]
@jophab Thanks for the review! Assuming you are listing according to popularity/most used commands, I'd suggest to swap `Marking Text (Visual Mode) and Search and Replace` sections as `search/replace `section is more commonly used on a daily basis by a programmer (myself being a nvim user :smile: ). What would you say?

**Comment by jophab** [comment]
@NaveenKarippai I was talking about the capitalization of first letter in each word.
You can choose the  ordering of sections  :+1:

**Comment by NaveenKarippai** [comment]
@jophab ah,true! sorry, I should have done a diff check :+1: 
Thanks again!

**Comment by zekiel** [comment]
looks good for merge! If there are issues or improvements, they will surface after getting more eyes on it via production. 

nice update @NaveenKarippai !

File: share/goodie/cheat_sheets/json/vim.json [hunk 1]
```diff
@@ -1,38 +1,48 @@
...
+        "Basic Configuration"
     ],
     "sections": {
         "Tabs": [{
             "val": "move to tab number #",
             "key": "#gt"
         }, {
             "val": "move the current split window into its own tab",
-            "key": "[Ctrl] [wt]"
+            "key": "[^w] [t]"
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/vim.json]: We should be using `Ctrl`, not `^` (that generally has a different meaning).

File: share/goodie/cheat_sheets/json/vim.json [hunk 2]
```diff
@@ -209,27 +237,57 @@
...
+            "key": "[^w] [_]"
         }, {
-            "val": "move cursor to next buffer (right)",
-            "key": "[Ctrl] [wh]"
+            "val": "minimise height of active window",
+            "key": "[^w] [1_]"
         }, {
-            "val": "move cursor to previous buffer (left)",
-            "key": "[Ctrl] [wl]"
+            "val": "equalise size of windows",
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/vim.json]: `Equalize`

File: share/goodie/cheat_sheets/json/vim.json [hunk 3]
```diff
@@ -209,27 +237,57 @@
...
+        }, {
+            "val": "minimise width of active window",
+            "key": "[^w] [1|]"
+        }, {
+            "val": "maximise height of active window",
+            "key": "[^w] [_]"
         }, {
-            "val": "move cursor to next buffer (right)",
-            "key": "[Ctrl] [wh]"
+            "val": "minimise height of active window",
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/vim.json]: `Minimize`

File: share/goodie/cheat_sheets/json/vim.json (hunk 1 above)
Comment (NaveenKarippai) [inline share/goodie/cheat_sheets/json/vim.json]: @GuiltyDolphin Thanks! I can make the changes. How should I proceed as PR has already been merged? **Restore the deleted branch and push new commit objects to the same should be good?**

Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/vim.json]: @NaveenKarippai Create a new branch with a new PR.

Comment (NaveenKarippai) [inline share/goodie/cheat_sheets/json/vim.json]: @GuiltyDolphin done! Please review PR https://github.com/duckduckgo/zeroclickinfo-goodies/pull/3170

**Pull request merged**